features/project-miniplayer/data.json

```
@@ -8,8 +8,8 @@
   "tags": ["New", "Featured"],
   "dynamic": true,
   "options": [
-    { "id": "position-right", "name": "Place the player to the right.", "type": 1 },
-    { "id": "position-bottom", "name": "Place the player to the bottom.", "type": 1 },
+    { "id": "position-right", "name": "Move the player to the right.", "type": 1 },
```

Player is probably enough but
```suggestion
{ "id": "position-right", "name": "Move the project player to the right.", "type": 1 },
```